certificate/authorization/db_test.go

```diff
@@ -154,38 +176,23 @@ func TestAuthorizationDB_Get(t *testing.T) {
 	authsBytes, err := expectedAuths.Marshal()
 	require.NoError(t, err)
 
-	err = authDB.db.Put(ctx, storage.Key("user@mail.test"), authsBytes)
+	err = authDB.db.Put(ctx, storage.Key("user@mail.example"), authsBytes)
```

Comment:
Not sure why this was changed but either `.test` or `.example` can be used https://tools.ietf.org/html/rfc2606#section-2.

* `.test` is recommended for tests.
* `.example` is recommended for examples.

Reply:
I keep forgetting and I've seen . example more often so my itch for consistency tingled. Will revert.

Reply:
`.test"` exists 76 times in our codebase, `.example"` 4 times.

Reply:
Interesting fact. I wasn't asserting example is used more, just that I'd seen it more. I guess I should read more 😛.

Reply:
Yeah, I know :D... all the ".example" things were in certificate code :D.